lighthouse-core/gather/driver.js

```diff
@@ -596,7 +596,10 @@ class Driver {
       .then(_ => this.sendCommand('Tracing.start', tracingOpts));
   }
 
-  endTrace() {
+  /**
+   * @param {number=} pauseAfterTraceMs Wait this many milliseconds before ending the trace
+   */
+  endTrace(pauseAfterTraceMs = 500) {
```

Comment:
dont you want a default of 0, since you're setting the config specifically for this pass?